docs/administration-guide/operation-score.md

```diff
@@ -0,0 +1,178 @@
+# Operation Scores
+
+Operation Scores can be used to give an OperationChains and NamedOperations a "score" which can then be used to determine whether a particular user has the required permissions to execute a given OperationChain.
+
+## Using Operation Scores
+
+A `ScoreOperationChain` operation determines a "score" for an OperationChain.
+
+Below is a simple example for constructing a ScoreOperationChain, using this directed graph:
+
+``` mermaid
+graph TD
+  1(1, count=3) -- count=3 --> 2
+  1 -- count=1 --> 4
+  2(2, count=1) -- count=2 --> 3
+  2 -- count=1 --> 4(4, count=1)
+  2 -- count=1 --> 5(5, count=3)
+  3(3, count=2) -- count=4 --> 4
+```
+
+!!! example "Example ScoreOperationChain"
+
+    === "Java"
+        ``` java
+        final ScoreOperationChain scoreOpChain = new ScoreOperationChain.Builder()
+            .operationChain(new OperationChain.Builder()
+                    .first(new GetElements())
+                    .then(new NamedOperation.Builder<Element, Iterable<? extends Element>>()
+                            .name("namedOp")
+                            .build())
+                    .then(new Limit<>(3))
+                    .build())
+            .build();
+        ```
+
+    === "JSON"
+        ``` json
+        {
+            "class" : "ScoreOperationChain",
+            "operationChain" : {
+                "class" : "OperationChain",
+                "operations" : [ {
+                    "class" : "GetElements"
+                }, {
+                    "class" : "NamedOperation",
+                    "operationName" : "namedOp"
+                }, {
+                    "class" : "Limit",
+                    "resultLimit" : 3,
+                    "truncate" : true
+                } ]
+            }
+        }
+        ```
+
+    === "Python"
+        ``` python
+        g.ScoreOperationChain( 
+            operation_chain=g.OperationChain( 
+                operations=[ 
+                    g.GetElements(), 
+                    g.NamedOperation( 
+                        operation_name="namedOp" 
+                    ), 
+                    g.Limit( 
+                        result_limit=3, 
+                        truncate=True 
+                    ) 
+                ] 
+            ) 
+        )
+        ```
+
+The key variables when using ScoreOperationChain are:
+
+- `opScores` - required map of operation scores. These are the operation score values.
```

Comment:
This is maybe a bit confusing as these keys are for the handler class not the `ScoreOperationChain` class itself as that technically only needs an `OperationChain` as input.

It might be worth breaking this section up to make it clear that this is the configuration for the default handler which you configure via the operations declarations file.

Reply:
I agree. These are variables used when configuring how Operation Scores are used with a graph, not when using the `ScoreOperationChain` Operation.